### ISSUE TITLE
[Bug]: Video Flickerig

### ISSUE THREAD

**Issue by InfinityAX** [FreeTubeApp/FreeTube#5593]
### Guidelines

- [X] I have encountered this bug in the [latest release of FreeTube](https://github.com/FreeTubeApp/FreeTube/releases).
- [X] I have encountered this bug in the [official downloads of FreeTube](https://github.com/FreeTubeApp/FreeTube#official-downloads).
- [X] I have searched the issue tracker for [open](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aopen+is%3Aissue) and [closed](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+is%3Aclosed) issues that are similar to the bug report I want to file, without success.
- [X] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the bug I want to file, without success.
- [X] This issue contains only one bug.

### Describe the bug

Intense flickering when playing any video. Sometimes, it just shows a black screen.

![Screenshot 2024-08-20 190147](https://github.com/user-attachments/assets/fd2ef946-6f1e-41bc-afa4-1fd6cbe648a7)

https://github.com/user-attachments/assets/e0dc4f17-4903-4656-99c8-e14e0a2c1beb

### Expected Behavior

Normal video playback with no flickering.

### Issue Labels

accessibility issue, visual bug

### FreeTube Version

v0.21.3 Beta

### Operating System Version

Windows 10 version 22H2

### Installation Method

.exe

### Primary API used

Local API

### Additional Information

GPU: Nvidia GeForce GTX 1650
Display: AOC 24G2W1G4

### Nightly Build

- [ ] I have encountered this bug in the latest [nightly build](https://docs.freetubeapp.io/development/nightly-builds).

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Please update your graphics drivers. If that doesnt resolve this check your graphics and display settings.

**Comment by InfinityAX** [comment]
Hello, I decided to switch to Linux, and the issue is no more. Something about the Windows driver seems to freak out my GTX 1650 (and yes, I tried the latest driver, but it did not work). I should have gone AMD.
